src/modules/kvs/cache.c

```diff
@@ -62,6 +62,7 @@ struct cache_entry {
                              * zero length data can be valid */
```

Comment:
This is a comment on the commit message (wish github would let us attach it in the right place!)

Use modules/kvs: prefix not src/modules/kvs.